Frameworks/UIKit/XamlControls.h

```diff
@@ -121,6 +121,12 @@ WXCGrid* CreateLabel();
 WXCTextBlock* GetLabelTextBlock(WXCGrid* labelGrid);
 
 ////////////////////////////////////////////////////////////////////////////////////
+// ScrollViewer
+////////////////////////////////////////////////////////////////////////////////////
+WXCGrid* CreateScrollViewer();
+WXCScrollViewer* GetScrollViewer(WXCGrid* labelGrid);
```

Comment:
actually, this is get the scrollViewer from the Grid. So there isn't confusion here.

---
In reply to: [106713268](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106713268) [](ancestors = 106713268,106712808)